src/main/java/edu/ucdavis/dss/ipa/api/components/budget/views/BudgetExcelView.java

```diff
@@ -155,7 +163,7 @@ protected void buildExcelDocument(Map<String, Object> model, Workbook workbook,
                                 sectionGroupCost.getEnrollment(),
                                 currentEnrollment,
                                 sectionGroupCost.getSectionCount(),
-                                (sectionGroupCost.getInstructor() == null ? "" : sectionGroupCost.getInstructor().getFullName()),
+                                String.join(", ", instructors),
```

Comment:
It looks like previously (before the multiple budget scenario download work) the Excel download included the instructor type assignments under the Instructor(s) column e.g. "Continuing Lecturer", "Associate Instructor"

I don't know if we want to fix it in this PR or make a new one.

Reply:
This PR is fine. Good catch.

Reply:
sounds good, I'll add that in.

Reply:
I tried it again and it doesn't look like it's picking up the instructor type descriptions?

Reply:
hmm yeah forgot to take into account teaching assignments without instructor costs.  My change only fixed it for non live data or live data if the cost is set explicitly.